From: Simon McVittie <smcv@debian.org>
Date: Fri, 24 Jun 2016 21:48:28 +0100
Subject: Downgrade failure to get document portal from warning to message

Otherwise, the tests all abort when run without FUSE.

Signed-off-by: Simon McVittie <smcv@debian.org>
Applied-upstream: 0.6.7, commit:https://github.com/flatpak/flatpak/commit/58bd3e3ec8e348b08ce4ce3cb339d42f0d49d197
---
 common/flatpak-run.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/flatpak-run.c b/common/flatpak-run.c
index 3c06371..d72f270 100644
--- a/common/flatpak-run.c
+++ b/common/flatpak-run.c
@@ -2481,7 +2481,7 @@ add_document_portal_args (GPtrArray  *argv_array,
         {
           if (g_dbus_message_to_gerror (reply, &local_error))
             {
-              g_warning ("Can't get document portal: %s\n", local_error->message);
+              g_message ("Can't get document portal: %s\n", local_error->message);
             }
           else
             {
